lib/invoiced/subscription.rb

```diff
@@ -10,5 +10,11 @@ class Subscription < Object
         def cancel
             delete
         end
+
+        def preview(params={}, opts={})
```

Comment:
I have a similar comment here as the PHP library about returning a JSON object, i.e. https://github.com/Invoiced/invoiced-ruby/blob/master/lib/invoiced/customer.rb#L18

Reply:
Resolved in same manner as PHP library